build.gradle.kts

```diff
@@ -65,6 +65,8 @@ tasks.processResources {
 
 allprojects {
     tasks.withType<Jar>().configureEach {
+        // Removing the version from the JARs here, since JARs in `profilerPlugins` are referenced by name in production code.
```

Comment:
I'd probably do that renaming on the consumer side. The producing project really shouldn't care about how it is consumed.

Alternatively, the consumer can have its version number in a manifest/properties file and use that to look up the file.

Reply:
I am renaming now on the consumer side.